packages/bitcore-node/test/benchmark/benchmark.ts

```diff
@@ -7,7 +7,11 @@ import { Storage } from '../../src/services/storage';
 import { BlockModel } from '../../src/models/block';
 import { BitcoinBlockType } from '../../src/types/namespaces/Bitcoin/Block';
 import { resetDatabase } from "../helpers/index.js";
+import * as crypto from 'crypto';
 
+function randomBytes() {
```

Comment:
I think `randomString()` might be more expressive here. Btw, any reason for the indentation missing?

Reply:
Thanks, I'll change that.

I may not have auto-formatted, I'll see.